cvat-ui/src/components/email-confirmation-page/email-confirmed.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Layout, Row, Col } from 'antd';
+
+const { Content } = Layout;
+
+/**
+ * Component for displaying email confirmation message and then redirecting to the loginpage
+ */
+class EmailConfirmationMessage extends React.Component {
+    componentDidMount() {
+        let counter = 5;
+        const id = setInterval(() => {
+            counter--;
+            document.getElementById('countID').innerHTML = counter;
+
+            if (counter === 0) {
+                try {
+                    document.getElementById('link').click();
```

Comment:
better to use refs if possible. Draft code is:

```
const linkRef = useRef<Link>();

function onFinish() {
    ref.current?.click();
}

return (
  ....
    <Link ref={linkRef} ... />
) 
```

Reply:
sure, I've changed to useRef